Sources/FeaturevisorSDK/DatafileReader.swift

```diff
@@ -32,18 +32,10 @@ public class DatafileReader {
     }
 
     public func getSegment(_ segmentKey: SegmentKey) -> Segment? {
-        let segment = self.segments.first(where: { $0.key == segmentKey })
-
-        // @TODO: parse conditions if stringified
-
-        return segment
+        return segments.first(where: { $0.key == segmentKey })
```

Comment:
maybe also remove `return` 😈

Reply:
Didn't want to go so far :D

Reply:
There will be more general cleaning around these stuff